src/App.tsx

```diff
@@ -1,18 +1,46 @@
-import React from 'react';
 import './App.css';
+import React, { useState } from 'react';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [total] = useState(items);
```

Comment:
We don't have to keep this in state as we never change it, it is static value

```suggestion
  const total = items.length;
```